<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Scientists</string>
       </attribute>
       <widget class="QListWidget" name="list_programmers">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>583</width>
          <height>281</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>351</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Add Scientist</string>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Name: </string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Birth:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Death: </string>
            </property>
           </widget>
           <widget class="QLineEdit" name="input_scientist_name">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>271</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="input_scientist_birth">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>60</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="input_scientist_death">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>90</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="button_add_scientist">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>130</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="button_scientist_male">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>60</y>
              <width>51</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="button_scientist_female">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>60</y>
              <width>61</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_scientist_error">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>211</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>231</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Filter</string>
           </property>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>181</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Ascending</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_6">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>70</y>
              <width>81</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Descending</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_5">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>130</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>490</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Seletect a Scientist to remove from database and push remove</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>351</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Add Computers</string>
           </property>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Name: </string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="input_computer_name">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>271</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="input_computer_year">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>60</y>
              <width>271</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="button_add_computer">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>130</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
           <widget class="QComboBox" name="dropdown_computer_type">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>90</y>
              <width>111</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>90</y>
              <width>61</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Was it built ?</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_computer_built_yes">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>90</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_computer_built_no">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>90</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>No</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>110</y>
              <width>231</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_computer_error">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>125</y>
              <width>171</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>229</width>
          <height>159</height>
         </rect>
        </property>
        <property name="title">
         <string>Filter</string>
        </property>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Ascending</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_10">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Descending</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>130</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </widget>
       <widget class="QListWidget" name="list_computers">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>583</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Seletect a Scientist to remove from database and push remove</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>490</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Relations</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>291</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>299</x>
          <y>129</y>
          <width>281</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QListView" name="listView_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>490</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Scientists</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Comptuers</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Search in scientists</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Search in computers</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>80</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>470</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select what you want to connect</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>291</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>Sort scientists</string>
           </property>
           <widget class="QComboBox" name="comboBox_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>131</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_11">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>20</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>ASC</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_14">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>20</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>DESC</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>281</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_11">
           <property name="title">
            <string>Sort computers</string>
           </property>
           <widget class="QComboBox" name="comboBox_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>131</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_12">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>20</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>ASC</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_13">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>20</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>DESC</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>9</y>
          <width>321</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="title">
            <string>Search in database</string>
           </property>
           <widget class="QLabel" name="label_16">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_13">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>261</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_11">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>70</y>
              <width>101</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>129</y>
          <width>581</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListView" name="listView_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>319</x>
          <y>9</y>
          <width>261</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="title">
            <string>Filter</string>
           </property>
           <widget class="QComboBox" name="comboBox_6">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>20</y>
              <width>121</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>111</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Scientist or computer:</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
